docs/reference/exceptions.md

```diff
@@ -0,0 +1,29 @@
+# Exceptions
+
+When creating pages with PyScript, you may encounter exceptions. Each handled exception will contain a specific code which will give you more information about it.
+This reference guide contains the error codes you might find and a description of each of them.
+
+## User Errors
+
+| Error code | Description                    | Recommendation     |
+|------------|--------------------------------|--------------------|
+| PY0001     | Feature is deprecated          | Check the documentation for the feature you are using and see recommendations. |
+| PY1000     | Invalid configuration supplied | Confirm that your `py-config` tag is using a valid `TOML` or `JSON` syntax and is using the correct configuration type. |
+
+
+
+## Fetch Errors
+
+These error codes are related to any exception raised when trying to fetch a resource. If, while trying to fetch a resource, we encounter a status code that is not 200, the error code will contain the HTTP status code and the `PY2` prefix. For example, if we encounter a 404 error, the error code will be `PY2404`.
```

Comment:
I chose the `PY` prefix mostly because that's our logo haha then the number after is referred to the type of error.

The idea is to have something like:
- PY0 (deprecated and dev one which should be changed to a proper error code)
- PY1 (any configuration issue)
- PY2 (any fetch issue)

As we expand the exceptions we could create different buckets so its easier to know where that error occurred from looking at it 😄